client/src/legacy/createEvent.js

```diff
@@ -0,0 +1,20 @@
+/* global window */
+// Polyfill IE11
+function createEvent(type, extraData) {
+  const { document, Event } = window;
+  let event;
+  if (typeof Event === 'object') {
+    event = document.createEvent('Event', true, true);
+    event.initEvent(type);
+  } else {
+    event = new Event(type);
+  }
+  if (extraData) {
+    Object.keys(extraData).forEach((key) => {
```

Comment:
`forEach(key =>`

Reply:
🤔 Interesting, I'm sure we did this because the linter was complaining, however it doesn't seem to be the case. It does bring a question of code style and consistency into play though...
E.g. I think the inclusion of parentheses makes `someCall('firstParam', firstParam => {...})` a little more readable.